editor.planx.uk/src/routes/views/flowEditor.tsx

```diff
@@ -144,3 +163,45 @@ export const flowEditorView = async (req: NaviRequest) => {
     </FlowEditorLayout>
   );
 };
+
+const includedServices = [
+  getFOIYNPP(environment),
+  getRAB(environment),
+  getSubmission(environment),
+  getDiscretionary(environment),
+];
+
+export const generateAnalyticsLink = ({
+  environment,
+  flowId,
+  flowSlug,
+}: {
+  environment: Environment;
+  flowId: string;
+  flowSlug: string;
+}): string | undefined => {
```

Comment:
I guess the `?? null` above is required as the signature of this returns `undefined`.

You can update the type of `flowAnalyticsLink` in the store to be `string | undefined` which might simplify things here 👍